Checkmate2019/Base/migrations/0001_initial.py

```diff
@@ -0,0 +1,41 @@
+# Generated by Django 2.1.4 on 2018-12-22 03:16
```

Comment:
@ashryaagr, @siddhantkhandelwal: Please do not commit and merge migrations. It will result in un-resolvable merge conflicts!

Reply:
Never faced issues. Thanks for pointing out though!
@ashryaagr Add them to .gitignore, remove exisiting ones.